microsoft-edge/extensions-chromium/publish/aad-account.md

```diff
@@ -1,75 +1,145 @@
 
-In order to add and manage account users, you must first associate your account with your organization's Azure Active Directory. If your organization already uses Office 365 or other business services from Microsoft, you already have Azure AD. Otherwise you can create a new Azure AD tenant for free. For more information, navigate to [Associate Azure Active Directory with your Partner Center account | Microsoft Docs][AssociateAzureADPCnew].
+1. Use the Microsoft account (MSA) which corresponds with your Partner Center developer account to sign in to your Partner Center developer account.
 
-Navigate to [Associate Azure Active Directory with your Partner Center account][AssociateAzureADPC] for more information. While the topic focuses on the Windows apps developer program, associating a tenant works the same way for the Microsoft Edge Program.
+1. Navigate to **Account settings** > [User management][UserMGMT] at Partner Center.
+
+1. Find out who is the Primary Owner of the Partner Center developer account.
+
+1. Verify that the Primary Owner of the Partner Center developer account is a Microsoft account (MSA) user.  This must not be a work Microsoft account (MSA) or a school Microsoft account (MSA).
+
+1. Have the Primary Owner of the Partner Center developer account register for the Microsoft Edge program by following the steps in [Register as a Microsoft Edge extension developer][DeveloperRegistration]<!-- = create-dev-account.md-->.
+
+Next, do [Step 2: Associate Azure Active Directory with your Microsoft Edge Program account](#step-2-associate-azure-active-directory-with-your-microsoft-edge-program-account) below.
+
+
+<!-- ====================================================================== -->
+## Step 2: Associate Azure Active Directory with your Microsoft Edge Program account
+
+Next, you'll link your Azure Active Directory tenants (Azure AD tenants) with your Microsoft Edge Program account, to enable managing Microsoft Edge extensions.  You can use Azure Active Directory to add users to your Microsoft Edge Program account and manage those users in that account.  You can add individual users, groups of users, or Azure Active Directory applications.
+
+To be able to add users to your Microsoft Edge Program account, and to manage those users in that account, you must first associate your Microsoft Edge Program account with your organization's Azure Active Directory tenant (Azure AD tenant).  If your organization already uses Office 365 or other business services from Microsoft, you already have an Azure AD tenant.  Otherwise, you can create a new Azure AD tenant for free.
+
+For more information, see the section [Create a brand new Azure AD to associate with your Partner Center account][AssociateAzureADPCnew] in the article _Associate Azure Active Directory with your Partner Center account_.
```

Comment:
0This sentence "for more information see the section Create a Brand new azure ad ...." is with reference to only those people who need to create a new AD tenant (in the previous sentence). Hence this should not be a separate sentence. At the moment, it seems like this step is necessary to go further, which is not true for those users who already have an AD tenant.